### PR TITLE
Add cancellation time to PlayStore subscription purchase object

### PR DESCRIPTION
**Pull request by SergeyKishenin**

### CODE REVIEW

**Comment by jnbt** [comment]
LGTM. 

@christophweegen Can you also have a look.

File: spec/play_store/subscription_purchases/subscription_purchase_spec.rb [hunk 1]
```diff
@@ -109,6 +109,34 @@
...
+        payment_state: 1,
+      )
+    end
+
+    it "is canceled?" do
+      subject.canceled_by_user?.must_be_true
+    end
+
+    it "returns the user_cancellation_time_millis" do
+      subject.user_cancellation_time_millis.must_equal 146_187_288_800_0
```
Comment (christophweegen) [inline spec/play_store/subscription_purchases/subscription_purchase_spec.rb]: I would suggest starting the integer grouping from the right. That trailing zero looks strange :-) 
```suggestion
      subject.user_cancellation_time_millis.must_equal 1_461_872_888_000
```

File: spec/play_store/subscription_purchases/subscription_purchase_spec.rb [hunk 2]
```diff
@@ -109,6 +109,34 @@
     end
   end
 
+  describe "canceled subscription by user" do
```
Comment (christophweegen) [inline spec/play_store/subscription_purchases/subscription_purchase_spec.rb]: Maybe you could add another test which confirms that `user_cancellation_time_millis` and `cancelled_at` return `nil`, in case the subscription wasn't canceled by the user. The reason why I think this could be important is because this `nil` could potentially be used for further conditionals, which would break if something "truthy" would be returned instead.

**Comment by SergeyKishenin** [comment]
@christophweegen changes were made

**Comment by jnbt** [comment]
@christophweegen Can we merge?

File: spec/play_store/subscription_purchases/subscription_purchase_spec.rb [hunk 3]
```diff
@@ -109,6 +109,61 @@
...
+          developer_payload: "payload that gets stored and returned",
+          payment_state: 1,
+        )
+      end
+
+      it "is not canceled?" do
+        subject.canceled_by_user?.must_be_false
+      end
+
+      it "returns blank user_cancellation_time_millis" do
```
[the remark this answers is not in the document]
Comment (christophweegen) [reply spec/play_store/subscription_purchases/subscription_purchase_spec.rb:129]: @jnbt @SergeyKishenin sorry for my late reply on that. Had made this request already back then, but since I never worked with the GitHub code review feature, I made a mistake and didn't submit it correctly. It was shown in my UI but I realized it wasn't public yet.

**Comment by jnbt** [comment]
@christophweegen Can your perform your last request yourself and merge it?
